yarn.lock

```diff
@@ -2681,6 +2690,22 @@
     "@polkadot/util" "^9.4.1"
     rxjs "^7.5.5"
 
+"@polkadot/api-derive@8.1.1":
```

Comment:
I don't think version 8.1.1 should be added to the yarn.lock

Reply:
yeah after doing a refresh install after yarn.lock removal this is right, although the resolution variation is taking over but let's do what you recommended